### DISCUSSION TITLE
Option to give a different name to the attached file

### DISCUSSION THREAD

**Discussion by paolo-santucci** [akosbalasko/yarle#197] [Ideas]
Hi @akosbalasko, I’m just trying your import tool: it’s really good!

I’m using a Fujitsu ScanSnap with Evernote and never bothered to rename the PDF files, so they’re a mess! Could you please add an option to save the attached file with the same name as the note?

**Comment by akosbalasko** [comment]
Hi @perfidopelato !
Sure thing, but please note that we have to cover the case when a note contains multiple attachments, so I would add an index to the file name as well. For example:
if ThisIsMyNote note
- contains 1 attachment named a.pdf, then it will be renamed to ThisIsMyNote.pdf 
- contains 2 attachments (a.pdf, b.pdf), then they will be renamed to  ThisIsMyNote.1.pdf and  ThisIsMyNote.2.pdf

How about this?

**Reply by paolo-santucci** [reply]
I think it’s perfect, thank you!

**Reply by akosbalasko** [reply]
Cool, okay, I implement it, thanks for the idea! 
I keep you updated.

**Reply by filmgal** [reply]
Has this been implemented? I just tried the Windows version of yarle today for the first time and came here to ask for that feature, as I couldn't see it anywhere in the app's UI options.

**Comment by grantx2016** [comment]
+1 This would be a useful bonus feature.
